tensorflow/core/kernels/conv_grad_ops.cc

```diff
@@ -888,8 +888,9 @@ class Conv2DSlowBackpropInputOp : public OpKernel {
       if (!blas_launch_status) {
         context->SetStatus(errors::Internal("Blas SGEMM launch failed : m=", m,
                                             ", n=", n, ", k=", k));
-        return;
       }
+
+      return;
```

Comment:
Could you do the one in conv_ops.cc as well? 

tensorflow/core/kernels/conv_ops.cc